android/ObjectPermissionAdvanced/app/src/main/java/io/realm/chat/util/PermissionHelper.java

```diff
@@ -0,0 +1,120 @@
+package io.realm.chat.util;
+
+import java.util.List;
+
+import io.realm.Realm;
+import io.realm.RealmResults;
+import io.realm.sync.permissions.ClassPermissions;
+import io.realm.sync.permissions.Permission;
+import io.realm.sync.permissions.PermissionUser;
+import io.realm.sync.permissions.RealmPermissions;
+import io.realm.sync.permissions.Role;
+import io.realm.chat.model.PrivateChatRoom;
+import io.realm.chat.ui.GrantedPermission;
+
+public class PermissionHelper {
```

Comment:
In general. This class could use some javadoc for the methods as it is kinda to the heart of this demo.